Gruntfile.js

```diff
@@ -320,6 +320,7 @@ module.exports = function(grunt) {
 				src: [
 					'src/manifest.json',
 					'src/data/*.json',
+					'!src/data/WhoCallsTheFleet*.json',

```

Comment:
oh, i forgot to mention: should rename extension of `WhoCallsTheFleet*` from `.json` to `.nedb` (as original author also done). because they are not standard json, they are: https://github.com/louischatriot/nedb
by renaming, will not need to exclude them from jsonlint, just add them to copy:processed session.